introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    tuning_job_config = {
</pre><p>Since this is meant to be an introductory notebook, it might be beneficial to add some basic exposition around how the tuning config is being defined here or even some links to HPO documentation to learn more outside of the notebook. To someone with no experience they might not even know where gamma, eta, etc. are coming from. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3623/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>